{"name": "Проверка возникновения ошибки при авторизации курьера без пароля", "status": "passed", "steps": [{"name": "Авторизация созданного курьера", "status": "passed", "parameters": [{"name": "login", "value": "'Ninja'"}, {"name": "password", "value": "''"}], "start": 1747235121551, "stop": 1747235121612}], "start": 1747235121551, "stop": 1747235121612, "uuid": "4bab0ba5-e6ad-4558-bf02-3a9afd1d59fc", "historyId": "4b888d15c89cf51a2090c58ae261518d", "testCaseId": "4b888d15c89cf51a2090c58ae261518d", "fullName": "tests.test_couriers.TestLoginCourier#test_login_courier_without_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_couriers"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-579T290"}, {"name": "thread", "value": "8216-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_couriers"}]}